compiler/rustc_resolve/src/imports.rs

```diff
@@ -769,9 +750,8 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                                 .emit();
                         }
                         let key = BindingKey::new(target, ns);
-                        this.update_resolution(parent, key, |_, resolution| {
-                            resolution.single_imports.remove(&Interned::new_unchecked(import));
-                        });
+                        let mut resolution = this.resolution(parent, key).borrow_mut();
+                        resolution.single_imports.remove(&Interned::new_unchecked(import));
```

Comment:
`update_resolution` has some extra logic related to `glob_importers`.
Does it never run in this case? Or it runs but it's irrelevant?

Reply:
It is a cleanup, because it seems will never meet other logic in `update_resolution` because the `binding` is not updated, and it will hit following code always:

```rust
 match resolution.binding() {
           _ if old_binding.is_some() => return t,
          None => return t,
      // xxxxxx
}
```


Of course, It should revert it back because it is irrrelative.